lib/node_modules/@stdlib/math/stats/kde2d/examples/index.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var normal = require( '@stdlib/random/base/normal' );
```

Comment:
Better, if possible, would be to plot the univariate densities, but this would require environment detection.